src/screens/Vault/AddSigningDevice.tsx

```diff
@@ -94,6 +106,15 @@ const onSignerSelect = (
       showToast('You have selected the total (n) keys, please proceed with the creation of vault.');
       return;
     }
+    if (scheme.m <= 3 && scheme.n <= 4) {
```

Comment:
How does this work here?

Reply:
It was working when we had 3-3 and 4-4 vaults so I added that one.

Reply:
Please ask Parsh to verify this one